src/v/security/audit/audit_log_manager.cc

```diff
@@ -27,6 +27,27 @@
 
 namespace security::audit {
 
+std::ostream& operator<<(std::ostream& os, event_type t) {
```

Comment:
nit: Maybe change the commit message because it seems like this commit is more then just adding log statements